eng/pipelines/templates/stages/archetype-sdk-management.yml

```diff
@@ -23,7 +23,7 @@ stages:
           JavaVersion: '1.8'
 
     pool:
-      vmImage: 'ubuntu-18.04'
```

Comment:
@weidongxu-microsoft are you guys good with moving your pipelines to ubuntu-20.04? 

Also we should chat about moving your pipelines to use our new 1ES Agent pools (see other changes in this PR with the name for those).

Reply:
Sure. I assume it going to follow the configure as `archtype-sdk-client.yml`?
```
- name: MatrixConfigs
  type: object
  default:
    - Name: Java_ci_test_base
      Path: eng/pipelines/templates/stages/platform-matrix.json
      Selection: sparse
      NonSparseParameters: Agent
      GenerateVMJobs: true
```

On the other hand, this template is used for track1 mgmt SDK. We haven't updated or released them for a couple of months. Though I guess for the sake of code reference, we still need to keep the code in main, but maybe we can schedule to skip the CI on them.